homework3/hw3/ViewController.swift

```diff
@@ -0,0 +1,93 @@
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class ViewController: UIViewController {
+    
+    private let topBackgroundView = UIView()
+    private let menuView = MenuView()
+    private let matchesTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.separatorStyle = .none
+        return tableView
+    }()
```

Comment:
```suggestion
    private let matchesTableView: UITableView = .init()
```

Dovoljno je ovako definirati, pa style dodaj u viewDidLoad ili style funkciji ako imas.